wp-google-login.php

```diff
@@ -27,12 +27,9 @@
 	return;
 }
 
-// If we don't have client id and secret then bail out, plugin won't work.
-if ( ! defined( 'WP_GOOGLE_LOGIN_CLIENT_ID' ) || ! defined('WP_GOOGLE_LOGIN_SECRET') ) {
-	return;
-}
 
 // We already making sure that file is exists and valid.
 require_once( sprintf( '%s/autoloader.php', WP_GOOGLE_LOGIN_PATH ) ); // phpcs:ignore
+require_once( sprintf( '%s/inc/functions.php', WP_GOOGLE_LOGIN_PATH ) ); // phpcs:ignore
```

Comment:
`require_once` is the statement and not the function so please remove `()` and also add the phpcs rule name that we need to ignore.  `// phpcs:ignore <rule-name>`

Reply:
Parentheses: Revised.
PHPCS: Not sure what the specific ignore rule would be. It appears to be ignoring on the first because autoloader doesn't exist until `composer install` is run. Not sure why it's included on the second.

Reply:
Revised. First it didn't want `sprintf()`, then it didn't want custom constant. Ignored custom constant at file level with `WordPressVIPMinimum.Files.IncludingFile.UsingCustomConstant`. According to https://github.com/squizlabs/PHP_CodeSniffer/issues/1179 specific rule ignores have to be at top of file.

Reply:
Revised again. No longer breaks to the PHPCS rule, so ignore not required.